src/hooks/onStartFetchData.ts

```diff
@@ -48,7 +54,10 @@ export default function getCacheLoadData() {
                   },
                 }
               );
-              console.log(user);
+              console.log(user.data.data.user?.recipes);
+              if (Array.isArray(user.data.data.user?.recipes)) {
+                dispatch(setUserRecipeListState(user.data.data.user?.recipes));
```

Comment:
Does this error out? we need for user.data.data.user.recipes to be a UserRecipe[] 

```js
// Core UserRecipe type
export type UserRecipe = {
  id: number;
  title: string;
  cuisine: string;
  dishType: string;
  vegetarian: Boolean;
  vegan: Boolean;
  glutenFree: Boolean;
  dairyFree: Boolean;
  readyInMinutes: number;
  servings: number;
  ingredients: string[];
  isSavored: Boolean;
};
```